OpenSim/Sandbox/MatlabScripts/c3d_reading.m

```diff
@@ -80,28 +79,24 @@
         rotatedData = [rotationMatrix'*vectorData']';
```

Comment:
This is actually an error. `rotationMatrix` should not be transposed. Would it be okay to just have `roatedData = [rotationMatrix*vectorData']';` ?